### PR TITLE
deploy: parameterize cephfs filesystem name

### PR DESCRIPTION
**Pull request by humblec**
This patch parameterize or make a variable for rook cephfs fs
name which we use in rook deployment

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: scripts/rook.sh [hunk 1]
```diff
@@ -4,6 +4,8 @@ ROOK_VERSION=${ROOK_VERSION:-"v1.4.9"}
 ROOK_DEPLOY_TIMEOUT=${ROOK_DEPLOY_TIMEOUT:-300}
 ROOK_URL="https://raw.githubusercontent.com/rook/rook/${ROOK_VERSION}/cluster/examples/kubernetes/ceph"
 ROOK_BLOCK_POOL_NAME=${ROOK_BLOCK_POOL_NAME:-"newrbdpool"}
+ROOK_FILESYSTEM_NAME=${ROOK_FILESYSTEM_NAME:-"myfs"}
```
Comment (Madhu-1) [inline scripts/rook.sh:7]: if you want to create a filesystem with this name, it needs to be patch/sed filesyste-test.yaml file then this will be useful

**Review by Madhu-1 — CHANGES_REQUESTED** [review]

File: scripts/rook.sh (hunk 1 above)
Comment (humblec) [reply scripts/rook.sh:7]: That can be choice, but the variable introduction with the default value is fine imo.

Comment (Madhu-1) [reply scripts/rook.sh:7]: but what is the use of a new variable? IMO the new variable makes sense if we are adding it to the filesystem-test.yaml

**Pull request closed without merging**